boards/nordic/nrf52840dk/src/main.rs

```diff
@@ -130,10 +145,16 @@ pub unsafe fn reset_handler() {
         LED2_PIN,
         LED3_PIN,
         led_pins,
+        &UartPins::new(UART_RTS, UART_TXD, UART_RXD, UART_CTS),
+        &SpiPins::new(SPI_MOSI, SPI_MISO, SPI_CLK),
+        &Some(SpiMX25R6435FPins::new(
+            SPI_MX25R6435F_DEVICE,
+            SPI_MX25R6435F_CLIENT,
```

Comment:
I was hoping you would fix these as well :)

Reply:
HAPPY NOW???!!!!!